lib/routemap.h

```diff
@@ -45,6 +48,12 @@ typedef enum {
 	RMAP_OKAY
 } route_map_result_t;
 
+#define RMAP_RESULT_STR(res) ((res == RMAP_MATCH) ? "match" : \
```

Comment:
So I would prefer we follow the standard we have throughout the code base for enum to strings and have a specialized function that has a switch statement ( see route_map_type_str() ) in that if we add a new value we will have a failed compile instead of having to `know` to edit this particular #define.  Especially since we don't follow this pattern any where else in the code base